ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__ocp4.15-lp-interop.yaml

```
@@ -108,10 +108,10 @@ tests:
       BASE_DOMAIN: cspilp.interop.ccitredhat.com
       FIREWATCH_CONFIG: |
         [
-         {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPINTEROP", "jira_additional_labels": ["4.15-lp","self-managed-lp","serverless-lp"]},
+         {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["4.15-lp","self-managed-lp","serverless-lp"]},
          {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "jira_project": "LPINTEROP", "jira_additional_labels": ["4.15-lp","self-managed-lp","serverless-lp"]},
          {"step": "gather-*", "failure_type": "all", "classification": "Other", "jira_project": "LPINTEROP", "jira_additional_labels": ["4.15-lp","self-managed-lp","serverless-lp"]},
-         {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "LPINTEROP", "jira_additional_labels": ["4.15-lp","self-managed-lp","serverless-lp"]},
+         {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["4.15-lp","self-managed-lp","serverless-lp"]},
```

```suggestion
         {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["4.15-lp","self-managed-lp","serverless-lp"]},
```

Caleb and I discussed this and found that this should never fail unless something is wrong with the ipi-install step. So in the case they both fail we want to only focus our attention on the priority 1 ipi-install.

---

Done in latest commit. Thank you both for this resolution